[2024-06-03T22:47:34.834+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T22:47:34.849+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [queued]>
[2024-06-03T22:47:34.854+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [queued]>
[2024-06-03T22:47:34.855+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T22:47:34.865+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-05-01 00:00:00+00:00
[2024-06-03T22:47:34.872+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=32880) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T22:47:34.874+0000] {standard_task_runner.py:63} INFO - Started process 32882 to run task
[2024-06-03T22:47:34.874+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-05-01T00:00:00+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp4811ipda']
[2024-06-03T22:47:34.875+0000] {standard_task_runner.py:91} INFO - Job 145: Subtask blunders_etl
[2024-06-03T22:47:34.909+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T22:47:34.975+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-01T00:00:00+00:00'
[2024-06-03T22:47:34.976+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T22:47:34.977+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T22:47:34.984+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T23:36:01.823+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T23:36:01.824+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T23:36:01.833+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-05-01T00:00:00+00:00, execution_date=20220501T000000, start_date=20240603T224734, end_date=20240603T233601
[2024-06-03T23:36:01.855+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T23:36:01.866+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T23:36:01.868+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T20:12:58.221+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T20:12:58.241+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [queued]>
[2024-06-08T20:12:58.251+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [queued]>
[2024-06-08T20:12:58.251+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T20:12:58.267+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-05-01 00:00:00+00:00
[2024-06-08T20:12:58.276+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=33151) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T20:12:58.278+0000] {standard_task_runner.py:63} INFO - Started process 33153 to run task
[2024-06-08T20:12:58.278+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-05-01T00:00:00+00:00', '--job-id', '321', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp9qocodgr']
[2024-06-08T20:12:58.279+0000] {standard_task_runner.py:91} INFO - Job 321: Subtask blunders_etl
[2024-06-08T20:12:58.323+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T20:12:58.405+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-01T00:00:00+00:00'
[2024-06-08T20:12:58.406+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T20:12:58.407+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T20:12:58.416+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T21:10:27.382+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T21:10:27.386+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T21:10:27.417+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-05-01T00:00:00+00:00, execution_date=20220501T000000, start_date=20240608T201258, end_date=20240608T211027
[2024-06-08T21:10:27.495+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T21:10:27.526+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T21:10:27.528+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T02:10:23.864+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T02:10:23.878+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [queued]>
[2024-07-08T02:10:23.883+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [queued]>
[2024-07-08T02:10:23.884+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T02:10:23.891+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-05-01 00:00:00+00:00
[2024-07-08T02:10:23.898+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=34170) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T02:10:23.899+0000] {standard_task_runner.py:63} INFO - Started process 34172 to run task
[2024-07-08T02:10:23.900+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-05-01T00:00:00+00:00', '--job-id', '717', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpjmjn2f3e']
[2024-07-08T02:10:23.900+0000] {standard_task_runner.py:91} INFO - Job 717: Subtask blunders_etl
[2024-07-08T02:10:23.936+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-05-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T02:10:23.999+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-01T00:00:00+00:00'
[2024-07-08T02:10:24.000+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T02:10:24.001+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T02:10:24.007+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T02:57:00.887+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T02:57:00.887+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T02:57:00.894+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-05-01T00:00:00+00:00, execution_date=20220501T000000, start_date=20240708T021023, end_date=20240708T025700
[2024-07-08T02:57:00.951+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T02:57:00.968+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T02:57:00.970+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
